src/Transpiler.ts

```diff
@@ -18,40 +18,15 @@ export class TranspileError extends Error {
 }
 
 export enum LuaTarget {
+    Lua51 = "5.1",
+    Lua52 = "5.2",
     Lua53 = "5.3",
     LuaJIT = "JIT",
 }
 
-export class LuaTranspiler {
+export abstract class LuaTranspiler {
     public static AvailableLuaTargets = [LuaTarget.LuaJIT, LuaTarget.Lua53];
```

Comment:
This isn't needed any more

Reply:
It prevents the class from being instantiated.